client/account-settings/components/label-payment-method/style.scss

```diff
@@ -0,0 +1,44 @@
+.card.is-compact.label-payment-method {
+	margin-bottom: 14px;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	cursor: pointer;
+
+	.payment-logo {
+		float: left;
+		margin-top: 0;
+		margin-right: 14px;
+	}
+}
+
+.label-payment-method__checkbox {
+	margin-right: 20px;
+	float: left;
+}
+
+.payment-logo {
```

Comment:
Do we need both this rule for `.payment-logo` and the one above it, for `.card.is-compact.label-payment-method .payment-logo`?